web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -162,6 +162,24 @@
               defer.reject();
               return $q.defer().promise;
             }
+            
+            //Check if the layer has some projection restriction
```

Comment:
This mechanic is not needed: Openlayers will reproject layers that are not in the current view projection on the fly

Reply:
But OL3 reprojection is not perfect. Sometimes it has weird white lines. Maybe you prefer to use a different layer to make sure it is shown correctly.

Reply:
I completely agree that the raster reprojection has an ugly glitch, and there's no way around it. But maybe I misunderstood your intent here: is the admin supposed to define several bg layers, each compatible with certain resolutions, to make sure the user always sees something clean whichever projection he/she's in?

Because then what would happen with layers added afterwards, ie by browsing metadata? The layer would simply disappear when changing proj? I mean, it could make sense but this would require giving a feedback to the user, for example in the layer panel ("error: this layer cannot be rendered in this proj" or something like that).

Sorry if my first understanding of your work was off. Let's see if someone else can review the PR.

Reply:
The idea is that you can define a layer using a projection restriction or not.

If there is a projection restriction it will be shown only on those projections.
If there is no projection restriction, it will be loaded always.

Layers added afterwards don't have a projection restriction so they will be loaded always (and OL3 handles reprojection).

Reply:
So this restriction only applies to base layers (the ones defined on the settings). Maybe I should document it better :)